cli-feed-v3.json

```diff
@@ -33,7 +33,7 @@
       "hidden": true
     },
     "v2-preview": {
-      "release": "2.29.0",
+      "release": "2.30.0",

```

Comment:
what is v2-preview?

Reply:
It seems like that was a tag Fabio added to release a beta version of Functions. I updated this as well just it in case. 🤷‍♂